Packs/HostIo/.secrets-ignore

```diff
@@ -3,4 +3,163 @@
 2607:f8b0:400a:803::200e
 172.217.14.0
 2607:f8b0:400a::
+getx2coinbasegiveaway.online
```

Comment:
why do you have so many secrets here?
* don't include duplicate values here (for example, `104.244.42.1` appears over 4 times)
* replace them in the README file (or where you have them) to already excluded items (for example, use the same 2-3 IP address instead of 10 addresses)